ui/app/templates/components/select.hbs

```diff
@@ -0,0 +1,25 @@
+
+  {{#if label}}
+    <label
+      for="select-{{this.elementId}}"
+      class="is-label"
+      data-test-select-label
+      >
+      {{label}}
+    </label>
+  {{/if}}
+  <div class="control {{if isInline "select is-inline-block"}}">
+    <div class="select {{if isFullwidth "is-fullwidth"}}">
+    <select
+      class="select"
+      id="select-{{this.elementId}}"
+      onchange={{action this.onChange value="target.value"}}
+      data-test-select={{name}}
+      >
+      {{#each options as |op|}}
+        <option value={{or (get op valueAttribute) op}} selected={{eq selectedItem op}}>{{or (get op labelAttribute) op}}</option>
```

Comment:
```suggestion
        <option value={{or (get op valueAttribute) op}} selected={{eq selectedItem (or (get op valueAttribute) op)}}>{{or (get op labelAttribute) op}}</option>
```
I think we'll need ^ for tracking the selected when `op` is an object.

Reply:
Hmm looking at this again, I'm not sure my suggestion is correct. Would `selectedItem` ever be an object? If not maybe it should be renamed `selectedValue`? If we do want to support passing objects, I think the onchange would have to change to support passing the selected object to the passed `onChange` function.

Reply:
`selectedItem` will never be an object since the `onchange` only passes the event `target.value`, so i've updated the attribute name to `selectedValue` and made a note in the docs. 😁